DM-4197 2024 LWM2M release note split device registration features (#2761)
SoftwareAG/c8y-docs: content/change-logs/device-management/cumulocity-10-18-288-0-LWM2M-PSK-generated-security-option.md

---
date: 2024-03-26T10:33:39.707Z

> title: PSK-generated option for LWM2M server security mode

Reviewer: ```suggestion
title:Auto-generated PSK option for LWM2M server security mode
```

Author: PSK-generated is the option visible in the UI. Therefore, will keep the title and desc. as is.

change_type:
  - value: change-QHu1GdukP
    label: Feature
product_area: Device management & connectivity
component:
  - value: component-1KLUzmqfe
    label: LWM2M
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: DM-554
version: 10.18.288.0
---

> During device registration, the "PSK generated" option can be selected for LWM2M Server connection to make LWM2M agent to generate the PSK ID and PSK key credentials for the device. These will be set to the device during the bootstrap process.

Reviewer 2: ```suggestion
During device registration, the option "PSK generated" can be selected for the LWM2M server connection. If this option is selected, the LWM2M agent generates the PSK ID and PSK key credentials for the device. These will be set to the device during the bootstrap process.
```

> During device registration, the option "PSK generated" can be selected for the LWM2M server connection. If this option is selected, the LWM2M agent generates the PSK ID and PSK key credentials for the device. These will be set to the device during the bootstrap process.

Reviewer: ```suggestion
During device registration, the option "PSK (auto-generated)" can be selected for the LWM2M server connection. If this option is selected, the LWM2M agent generates the PSK ID and PSK key credentials for the device. These will be set to the device during the bootstrap process.
```

Author: PSK-generated is the option visible in the UI. Therefore, will keep the title and desc. as is.

> For details, refer to [LWM2M security settings](/protocol-integration/lwm2m/#security-settings).

Reviewer: ```suggestion
For details, refer to the [LWM2M security settings](/protocol-integration/lwm2m/#security-settings).
```

> For details, refer to the [LWM2M security settings](/protocol-integration/lwm2m/#security-settings) user documentation.

Reviewer 2: ```suggestion
For details, refer to [LWM2M security settings](/protocol-integration/lwm2m/#security-settings).
```
